mofacts/client/views/experiment/card.js

```diff
@@ -2352,7 +2352,8 @@ function startUserMedia(stream) {
         }
         Router.go('/card');
         return;
-      } else if (!Session.get('recording')) {
+      } else if (!Session.get('recording') || Session.get('pausedLocks') != 0) {
```

Comment:
Minor quibble but can you make the condition > 0 instead of != 0?  != 0 could be true if undefined or we accidentally subtract below 0.